@here/harp-examples/example-browser.ts

```diff
@@ -50,22 +50,76 @@ function exampleBrowser(exampleDefinitions: ExampleDefinitions) {
         Object.keys(exampleDefinitions)
             .sort()
             .forEach(pageUrl => {
-                const linkName = getName(pageUrl);
+                const linkName = getName(pageUrl).replace(new RegExp("-", "g"), " ");
```

Comment:
Why not
```suggestion
                const linkName = getName(pageUrl).replace(/-/g, " ");
```
?

Reply:
Correct! However it is the only remaining change and I'd like not to retrigger the CI for it and merge the topic if you're ok with it.